kerastuner/engine/metrics_tracking.py

```diff
@@ -16,15 +16,23 @@
 from __future__ import division
 from __future__ import print_function
 
+import collections
 import copy
 import numpy as np
+from tensorflow import keras
+
+
+MetricObservation = collections.namedtuple(
+    'MetricObservation',
+    'value t')
```

Comment:
Can a MetricObservation please contain both steps and elapsed_sec (which I assume is what `t` is)?
Both are useful to know, especially in distributed tuning, where jobs may be preempted and warm-started from a checkpoint.